src/extractor/guidance/intersection_handler.cpp

```diff
@@ -341,15 +344,28 @@ bool IntersectionHandler::isThroughStreet(const std::size_t index,
 {
```

Comment:
This function could also use a small explanation. E.G. compares multiple roads at the intersection and determines if the `index` would be considered the best candidate for a `through` street.